tools/requirements.txt

```diff
@@ -0,0 +1,39 @@
+certifi==2022.12.7
+charset-normalizer==3.0.1
+click==8.1.3
+colorama==0.4.6
+filelock==3.9.0
+huggingface-hub==0.12.0
+idna==3.4
+joblib==1.2.0
+keybert==0.7.0
+markdown-it-py==2.1.0
```

Comment:
is this file something like `yarn.lock`? Is there a script I can run to install all these libraries at once?

Reply:
Yes, it is. you can run `pip install -r requirements.txt` to install all the dependencies :)

Reply:
> is this file something like `yarn.lock`? Is there a script I can run to install all these libraries at once?

Yes, it is. we can run `pip install -r requirements.txt` to install all the dependencies :)